var/spack/repos/builtin/packages/emacs/package.py

```diff
@@ -58,8 +77,18 @@ class Emacs(AutotoolsPackage, GNUMirrorPackage):
     depends_on('texinfo', type='build', when="@master:")
     depends_on('gcc@11: +strip languages=jit', when="+native")
 
+    depends_on('mailutils', when='+mailutils')
+    depends_on('krb5', when='+kerberos5')
+    depends_on('gmp')
+
+    depends_on('acl')
+    depends_on('imagemagick', when='+imagemagick')
+    depends_on('alsa-lib', when='sound=alsa')
+
+    depends_on('jansson', when='+json')
+
     conflicts('@:26.3', when='platform=darwin os=catalina')
-    conflicts('+native', when='@:27', msg="native compilation require @master")
+    conflicts('+native', when='@:27', msg="native compilation requires @master")
```

Comment:
Is there some future version that will support this, or is it only going to be supported on master going forward?

Reply:
Yes! I believe version 28 will! Great catch!